deploy/kubernetes/base/controller.yaml

```diff
@@ -126,7 +126,8 @@ spec:
               cpu: 10m
               memory: 40Mi
           securityContext:
-            allowPrivilegeEscalation: false
+            seccompProfile:
+              type: RuntimeDefault
```

Comment:
Is this guaranteed to exist on all Kubernetes installations? What if the OS the user is using has SELinux disabled?

Reply:
> Is this guaranteed to exist on all Kubernetes installations?

It is available in all supported EKS versions (including extended support). 

> What if the OS the user is using has SELinux disabled?

My understanding is that seccomp and SELinux operate independently. That is to say, if SELinux state is disabled (not enforcing policies), seccomp profiles can still be applied.

Reply:
:facepalm: this is about seccomp not selinux

Discussed offline, new version lgtm